### PR TITLE
fix: OrderEdit modal UI alignment

### PR DESCRIPTION
**Pull request by fPolic**
**What**
- polish OrderEdit modal UI to be aligned with Figma

**_BEFORE_**
Screenshot 2023-01-20 at 13 39 40

**_AFTER_**
Screenshot 2023-01-20 at 13 38 55

---

DEPENDS ON https://github.com/medusajs/medusa/pull/3108

---

FIXES CORE-763

### CODE REVIEW

File: src/domain/orders/details/order-line/edit.tsx [hunk 1]
```diff
@@ -250,32 +265,31 @@ const OrderEditLine = ({
...
+                className={clsx("text-gray-900 min-w-[60px] text-right", {
+                  "!text-gray-400 pointer-events-none": isLocked,
+                })}
+              >
+                {formatAmountWithSymbol({
+                  amount: item.unit_price * item.quantity,
+                  currency: currencyCode,
+                  tax: item.tax_lines,
+                  digits: 2,
+                })}
```
Comment (srindom) [inline src/domain/orders/details/order-line/edit.tsx:285]: There is something wrong with how this total is being calculated. I see taxes added twice. 

Do we have access to `item.total` which we typically respond with on the backend?

Comment (srindom) [reply src/domain/orders/details/order-line/edit.tsx:285]: Maybe we should add line item totals to order edit line items to make this simpler

Comment (fPolic) [reply src/domain/orders/details/order-line/edit.tsx:285]: > Do we have access to item.total which we typically respond with on the backend

We don't, should we open a ticket for the core to decorate OE items with totals as we do with order items?

Comment (srindom) [reply src/domain/orders/details/order-line/edit.tsx:285]: Yes will do it :)

**Comment by olivermrbl** [comment]
While we are at it, should we add the missing actions to the timeline events? Will create an issue for it

**Comment by olivermrbl** [comment]
@srindom Will you have time to give this another review today?

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**